src/applications/claims-status/components/NeedHelp.jsx

```diff
@@ -35,9 +35,9 @@ export function NeedHelp({ item }) {
     <va-need-help class="vads-u-margin-top--9">
       <div slot="content">
         <p>
-          Call us at <va-telephone contact="8008271000" />. We're here Monday
-          through Friday, 8:00 a.m to 9:00 p.m ET. If you have hearing loss,{' '}
-          <va-telephone contact="711" tty="true" />.
+          Call the VA benefits hotline at <va-telephone contact="8008271000" />.
+          We're here Monday through Friday, 8:00 a.m to 9:00 p.m ET. If you have
```

Comment:
@rallencf This is a question for you for sure - should the `a.m`/`p.m` tags on this ticket be formatted in this way?  I am seeing it is accurate to the Figma, but it looks strange to me 🤔 just want to confirm with you that this is intentional before merging!  I would have expected to see either `am`/`pm` (no punctuation) or `a.m.`/`p.m.` (both letters punctuated).

Reply:
@janechodance per @rallencf in Slack:
```
It should be a.m. and p.m., with periods. Thanks for checking! Not sure what happened to those couple of missing periods.
```